.github/workflows/test-and-deploy.yml

```diff
@@ -121,6 +97,71 @@ jobs:
         uses: sendgrid/dx-automator/actions/datadog-release-metric@main
         env:
           DD_API_KEY: ${{ secrets.DATADOG_API_KEY }}
+          
+  code-signing:
+    runs-on: windows-latest
+    needs: [ deploy ]
+    steps:
+      - name: Checkout sendgrid-csharp
+        uses: actions/checkout@v2
+
+      - name: Setup .NET Core SDK
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '3.1.x'
+
+      - name: Set up certificate
+        run: |
+          echo "${{ secrets.SM_CLIENT_CERT_FILE_B64 }}" | base64 --decode > /d/Certificate_pkcs12.p12
+          cat  /d/Certificate_pkcs12.p12
+        shell: bash
+
+      - name: Set variables
+        id: variables
+        run: |
+          dir
+          echo "::set-output name=version::${GITHUB_REF#refs/tags/v}"
+          echo "::set-output name=KEYPAIR_NAME::gt-standard-keypair"
+          echo "::set-output name=CERTIFICATE_NAME::gt-certificate"
+          echo "SM_HOST=${{ secrets.SM_HOST }}" >> "$GITHUB_ENV"
+          echo "SM_API_KEY=${{ secrets.SM_API_KEY }}" >> "$GITHUB_ENV"
+          echo "SM_CLIENT_CERT_FILE=D:\\Certificate_pkcs12.p12" >> "$GITHUB_ENV"
+          echo "SM_CLIENT_CERT_PASSWORD=${{ secrets.SM_CLIENT_CERT_PASSWORD }}" >> "$GITHUB_ENV"
+          echo "BUILD_TOOLS_VERSION=31.0.0" >> "$GITHUB_ENV"
+          echo "C:\Program Files (x86)\Windows Kits\10\App Certification Kit" >> $GITHUB_PATH
+          echo "C:\Program Files (x86)\Microsoft SDKs\Windows\v10.0A\bin\NETFX 4.8 Tools" >> $GITHUB_PATH
+          echo "C:\Program Files\DigiCert\DigiCert Keylocker Tools" >> $GITHUB_PATH
+        shell: bash
+
+      - name: Code signing with Software Trust Manager
+        id: SSMClientToolSetup
+        uses: digicert/ssm-code-signing@v0.0.2
+        env:
+          SM_API_KEY: ${{ env.SM_API_KEY }}
```

Comment:
Rather than setting the API_Key in env var , can we use it from Github secrets? I see from line 127 that its originally retrieved from secrets. 
Also, please make sure that you add details and values of these secrets in 1pass vault.

Reply:
Actually, I tried using SM_API_KEY from secrets but [here](https://github.com/twilio/twilio-csharp/blob/4ceb352d08e7dec4e0ddffa9ff609899bea8105d/.github/workflows/test-and-deploy.yml#L148) it reads env variables only, so for that we need to set the env variable